### DISCUSSION TITLE
Combined logo of HomeConnect and MQTT

### DISCUSSION THREAD

**Discussion by Oktonaut** [hcpy2-0/hcpy#64] [Ideas]
I once did the work of creating a logo... if you like it, you're welcome to use it freely ;-)

![hcpy-logo](https://github.com/hcpy2-0/hcpy/assets/109097757/d29fcfe0-9db6-46fb-bc2e-5d9bdb381791)

**Comment by pmagyar** [comment]
Hey @Oktonaut,

thanks for that great work. I like it and think it fully reflects the project.
I will be very happy to use it.
